feature-libs/product/configurators/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.spec.ts

```diff
@@ -0,0 +1,161 @@
+import { ChangeDetectionStrategy } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ConfiguratorAttributeMultiSelectionBundleComponent } from './configurator-attribute-multi-selection-bundle.component';
+import { Configurator } from '../../../../core/model/configurator.model';
+import { ConfiguratorAttributeProductCardComponent } from '../../product-card/configurator-attribute-product-card.component';
+import { I18nTestingModule } from '@spartacus/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { UrlTestingModule } from 'projects/core/src/routing/configurable-routes/url-translation/testing/url-testing.module';
```

Comment:
this import is incorrect and will break the build. import this correctly

Reply:
Module '"@spartacus/core"' has no exported member 'UrlTestingModule'.